src/meltano/cli/select.py

```diff
@@ -35,7 +35,7 @@ def selection_color(selection: SelectionType) -> str | None:
     if selection is SelectionType.UNSUPPORTED:
         return "black"
 
-    return None  # pragma: no cover
+    return None  # type: ignore[unreachable]  # pragma: no cover
```

Comment:
Same as above, if the type checker can't reach this, then I think we can remove the return statement, and update the return signature to only be `str`. Then, if we ever add a new `SelectionType`, we'll get a type error because there'll be a way to return `None` from a function that is only supposed to return `str`.

Reply:
4223e579f